source/plugins/wakatime/metadata.yml

```
@@ -53,3 +52,22 @@ inputs:
     type: number
     default: 5
     min: 0
+
+  # If you use a selfhosted wakatime instance
+  # that is publicly available set this to true
+  plugin_wakatime_wakapi:
+    description: Use WakaTime or self hosted Wakapi instance
+    type: boolean
+    default: no
+
+  # If you use the public wakapi instance the url 
+  # would be https://wakapi.dev
+  plugin_wakatime_apiurl:
+    description: Address where to reach your Wakapi instance
+    type: string
+    default: ""
+
+  plugin_wakatime_user:
+    description: Your Wakatime user on the selfhosted Wakapi instance
+    type: string
+    default: ""
```

```suggestion
    default: .user.login
```
`.user.login` will automatically be parsed as current user

---

ohh thats bigbrain